assets/src/components/v2/dup/dup_rotation_wrapper.tsx

```diff
@@ -0,0 +1,11 @@
+import _ from "lodash";
+import React from "react";
+
+export const splitRotationFromPropNames = (WrappedComponent: React.ElementType, rotation: string) => {
+  return (props: any) => {
+      
+    const modifiedProps = _.mapKeys(props, (_value, key) => key.replace(`_${rotation}`, ""))
```

Comment:
Super minor suggestion: a regex pattern could be a bit safer as you can specify that the rotation needs to appear at the end of the key string:
```suggestion
    const rotationSuffixPattern = new RegExp(`_${rotation}$`);
    const modifiedProps = _.mapKeys(props, (_value, key) => key.replace(rotationSuffixPattern, ""))
```

For example:
```js
const rot = "one";
// The '$' at the end of the pattern anchors it to the end of the string
const re = new RegExp(`_${rot}$`);

console.log( "foo_one".replace(re, "") );
> "foo"

// Does not get replaced since it's not at the end of the string
console.log( "foo_one_x".replace(re, "") );
> "foo_one_x"
```

Reply:
It doesn't hurt to be more precise! 👌